src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -176,6 +178,16 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
   /** New files that should be added to the list of target files for the next iteration. */
   private final Set<String> addedTargetFiles = new HashSet<>();
 
+  /** Stores the set of method names declared in the currently visiting class. */
+  private final Set<String> declaredMethod = new HashSet<>();
+
+  /**
+   * Maps the name of a class to the unsolved interface that it implements. If a class implements
+   * multiple unsolved interfaces, only the last one is tracked in this map.
```

Comment:
Why is only the last interface tracked? (I think the reason is that Specimin wouldn't be able to tell which interface declares which method anyway, but I'd like to be sure.)

Also, please write at least one test case with two unsolved interfaces, so that this behavior is tested.